app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly implement relationship between `User` and other models following the this [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)